packages/@aws-cdk/pipelines/README.md

```diff
@@ -100,45 +104,45 @@ to build. The Pipeline will be provisioned in account `111111111111` and region
 
 ```ts
 class MyPipelineStack extends Stack {
-  constructor(scope: Construct, id: string, props?: StackProps) {
-    super(scope, id, props);
-
-    const sourceArtifact = new codepipeline.Artifact();
-    const cloudAssemblyArtifact = new codepipeline.Artifact();
-
-    const pipeline = new CdkPipeline(this, 'Pipeline', {
-      pipelineName: 'MyAppPipeline',
-      cloudAssemblyArtifact,
-
-      sourceAction: new codepipeline_actions.GitHubSourceAction({
-        actionName: 'GitHub',
-        output: sourceArtifact,
-        oauthToken: SecretValue.secretsManager('GITHUB_TOKEN_NAME'),
-        trigger: codepipeline_actions.GitHubTrigger.POLL,
-        // Replace these with your actual GitHub project name
-        owner: 'OWNER',
-        repo: 'REPO',
-      }),
-
-      synthAction: SimpleSynthAction.standardNpmSynth({
-        sourceArtifact,
+    constructor(scope: Construct, id: string, props?: StackProps) {
```

Comment:
we use 2 spaces in our code-base (for TS), so I think we shouldn't change the spacing here

Reply:
I'm gonna blame VS Code for that... yeah